<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parameter</class>
 <widget class="QDialog" name="Parameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>319</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>363</width>
    <height>319</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>363</width>
    <height>319</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="TolConQual_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tolerance Conserved Quality</string>
   </property>
  </widget>
  <widget class="QLabel" name="k_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>k [s/cm]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="R">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>2.5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="theta0">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="export_button">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>280</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QLabel" name="theta0_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>theta0 [rad]</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>m [g]</string>
   </property>
  </widget>
  <widget class="QLabel" name="a_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a [cm]</string>
   </property>
  </widget>
  <widget class="QLabel" name="R_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R [cm]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TolConQual">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00001</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="a">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Friction">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Friction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="import_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QLabel" name="psidot_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>psidot0 [rad/s]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="psidot0">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>250</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="k">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="t_max_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>t_max [s]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="t_max">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>2.75</string>
   </property>
  </widget>
  <action name="actionSetButtonClicked">
   <property name="text">
    <string>SetButtonClicked</string>
   </property>
  </action>
  <action name="actionCancelButtonClicked">
   <property name="text">
    <string>CancelButtonClicked</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Friction</sender>
   <signal>clicked()</signal>
   <receiver>Parameter</receiver>
   <slot>FrictionCheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Submit</sender>
   <signal>clicked()</signal>
   <receiver>actionSetButtonClicked</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>actionCancelButtonClicked</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Submit()</slot>
  <slot>FrictionCheck()</slot>
 </slots>
</ui>
